core/sail/shacl/src/test/java/org/eclipse/rdf4j/sail/shacl/AbstractShaclTest.java

```diff
@@ -273,7 +295,7 @@ static void runTestCase(String shaclPath, String dataPath, ExpectedResult expect
 				connection.begin(isolationLevel);
 				ValueFactory vf = connection.getValueFactory();
 				connection.add(vf.createBNode(), vf.createIRI("http://example.com/jkhsdfiu3r2y9fjr3u0"),
-						vf.createLiteral("auto-generated!"), vf.createBNode());
+						vf.createLiteral("123", XMLSchema.INTEGER), vf.createBNode());
```

Comment:
I think this is what causes your current test failure - on the develop branch, the import for XMLSchema in this class has been replaced with XSD. CI tries to build the PR after merging, and fails because it misses the import. 

Easiest way to fix is to rebase and then replace this reference with XSD.

Reply:
Ahh. That makes sense. The import gets changed to XSD, but the reference XMLSchema.INTEGER can't resolve.